app/src/pages/[resourceType]/[id].tsx

```diff
@@ -73,6 +80,17 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
   );
 }
 
+/**
+ * Function which extracts specified content from JSON data and then returns the decoded
+ * version of that content (in this case the ELM/CQL)
+ * @returns The decoded version of the ELM/CQL content
+ */
+function Decode(link: string, jsonData: FhirArtifact) {
```

Comment:
Last suggestion is to make this function lowercase `decode`. Uppercase/PascalCase should be reserved for react components and classes